.kitchen.yml

```diff
@@ -36,7 +37,7 @@ default: &default
 suites:
   - name: default
```

Comment:
Yeah. Some suites need to run on certain platforms. 

Having all suites on all platforms is a no-go. But I would want to cut down the number of platforms we run each suite on. For e.g. run only one suite on ubuntu-1204 and other suite on ubuntu-1004. 

@patrick-wright thoughts on that?